package.json

```diff
@@ -28,18 +28,20 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "@babel/runtime": "7.22.15",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "@material-table/core": "^4.3.11",
     "@mov-ai/mov-fe-lib-code-editor": "^1.0.8-2",
     "@remixproject/engine": "^0.3.19",
     "@remixproject/engine-web": "^0.3.19",
+    "clsx": "^1.1.1",
     "d3": "^5.16.0",
     "hotkeys-js": "^3.8.7",
     "joi": "^17.4.2",
     "lodash": "^4.17.21",
-    "material-table": "^1.69.3",
     "monet": "^0.9.2",
+    "prop-types": "^15.8.1",
```

Comment:
Why add this library?

Reply:
Same as the above:

Because we are using it: ![2023-09-25-095518_1920x1080_scrot](https://github.com/MOV-AI/frontend-npm-lib-ide/assets/4276265/548fffaa-1e76-4b76-94c1-fc4d3f7c9fc2)

I don't know why it didn't complain before.

Reply:
:+1: